session_items.py

```diff
@@ -1,4 +1,5 @@
 from flask import session
+import requests
```

Comment:
You've added a dependency that needs to be installed before you can run the project. 

You should add it to requirements.txt, mainly so that other people can easily install it. After doing `pip install ...`, you can do `pip freeze` to see your installed packages.

Reply:
requests installed and added to requirements.txt